apps/s_client.c

```diff
@@ -2765,8 +2765,20 @@ int s_client_main(int argc, char **argv)
     }
 
     for (;;) {
+        int fdin, fdout;
+
         FD_ZERO(&readfds);
         FD_ZERO(&writefds);
+	fdin = fileno_stdin();
```

Comment:
There's a TAB here, we now always use spaces. Do we care?

Reply:
This appears to need a rebase. Perhaps the TAB can be expanded while rebasing?

Reply:
Yes.

Reply:
@vdukhovni are you going to take care of it, or should I put another commit on top that fixes it?

Reply:
> @vdukhovni are you going to take care of it, or should I put another commit on top that fixes it?

If you don't mind rebasing it, and expanding the TAB while you're at it, that'd be great...